pycyphal/transport/udp/_ip/_endpoint_mapping.py

```diff
@@ -120,28 +125,30 @@ def message_data_specifier_to_multicast_group(
 
     >>> from pycyphal.transport import MessageDataSpecifier
     >>> from ipaddress import ip_address
-    >>> str(message_data_specifier_to_multicast_group(ip_address('127.42.11.22'), MessageDataSpecifier(123)))
-    '239.42.0.123'
+    >>> str(message_data_specifier_to_multicast_group(ip_address('127.40.11.22'), MessageDataSpecifier(123)))
+    '239.40.0.123'
     >>> str(message_data_specifier_to_multicast_group(ip_address('192.168.11.22'), MessageDataSpecifier(456)))
     '239.40.1.200'
     >>> str(message_data_specifier_to_multicast_group(ip_address('239.168.11.22'), MessageDataSpecifier(456)))
     Traceback (most recent call last):
       ...
     ValueError: The local address shall be a unicast address, not multicast: 239.168.11.22
+    >>> msg_ip = message_data_specifier_to_multicast_group(ip_address('127.40.11.22'), MessageDataSpecifier(123))
+    >>> assert((int(msg_ip) & DATASPECIFIER_BIT_MASK) != DATASPECIFIER_BIT_MASK, "Dataspecifier bit is zero for message")
```

Comment:
Do you think this test makes sense?

![image](https://user-images.githubusercontent.com/39975120/196039256-80ab5440-d338-4dc4-b293-b5ce63a7bab3.png)

`pytest` complains about it always being true

Reply:
It is always true because you are asserting that a tuple of two elements is true, which it is, and the check is therefore useless. Remove the outer parentheses to fix it.